### PR TITLE
Prepare for Holochain 0.3

### PR DESCRIPTION
**Pull request by ThetaSinner**
See this issue #211 and this conversation https://discord.com/channels/727959104777224303/728647693244694648/1171849740850319370

### CODE REVIEW

File: src/api/client.ts [hunk 1]
```diff
@@ -188,6 +188,14 @@ export class WsClient extends Emittery {
     resolve: RequestResolver,
     reject: RequestRejecter
   ) {
+    function toHexString(byteArray: number[]) {
+      return Array.from(byteArray, function(byte) {
+        return ('0' + (byte & 0xFF).toString(16)).slice(-2);
+      }).join(', ')
+    }
+
+    console.log("sending message", request, toHexString(encode(request) as unknown as number[]))
```
Comment (ThetaSinner) [inline src/api/client.ts]: This is super useful for debugging, do not intend to it to be merged to `main`!

**Comment by ThetaSinner** [comment]
This is enough to get a `list-apps` admin API call working, the tests are completely broken

**Comment by jost-s** [comment]
Yeah groovy! No, I hadn't noticed that the ser/de had changed on the Holochain side. I'm grateful for you help. The tests weren't completely broken, only 2 of them weren't passing. The fix was the same as you did for serializing requests, but for deserializing responses. Should be good to go now.

**Comment by ThetaSinner** [comment]
Oh good! I ran them locally and they dumped out a lot of complaints so I assumed I'd caused some breakage. Great that it was closer than I thought :)

**Pull request merged**